specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "description": "Get a list of all private offers in the given private store",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OfferListResponse"
+            }
+          },
+          "404": {
+            "description": "No private store enabled"
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreOffers": {
+            "$ref": "./examples/GetPrivateStoreOffers.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers/{OfferId}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/OfferIdPathParameter"
+        },
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "delete": {
+        "description": "Deletes an offer from the given private store.",
+        "responses": {
+          "200": {
+            "description": "OK - offer deleted"
+          },
+          "404": {
+            "$ref": "#/responses/PrivateStoreOffer404"
+          }
+        }
+      },
+      "get": {
+        "description": "Gets information about a specific offer.",
+        "responses": {
+          "200": {
+            "description": "Offer information retrieved",
+            "schema": {
+              "$ref": "#/definitions/Offer"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreOffer": {
+            "$ref": "./examples/GetPrivateStoreOffer.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Update or add an offer to the default collection of the private store.",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/offerObjectParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
```

Comment:
>  "description": "OK" [](start = 10, length = 21)

PUT must have a body and should match GET

Reply:
done